cluster/addons/fluentd-elasticsearch/fluentd-es-configmap.yaml

```diff
@@ -4,6 +4,7 @@ metadata:
   name: fluentd-es-config-v0.2.0
   namespace: kube-system
   labels:
+    kubernetes.io/cluster-service: "true"
```

Comment:
Why is this added? As far as a know this label is deprecated.

Reply:
Yep, you are right. Removed it.